src/server/managers/AppWebhook.ts

```diff
@@ -0,0 +1,108 @@
+import { AppMethod } from '../../definition/metadata';
+
+import { IWebhook, IWebhookRequest, IWebhookResponse, WebhookSecurity, WebhookVisibility } from '../../definition/webhooks';
+import { ProxiedApp } from '../ProxiedApp';
+import { AppLogStorage } from '../storage';
+import { AppAccessorManager } from './AppAccessorManager';
+
+const methods: Array<string> = [
+    'get',
+    'post',
+    'put',
+    'delete',
+    'head',
+    'options',
+    'patch',
+];
+
+export class AppWebhook {
```

Comment:
What's the purpose of having two `AppWebhook` items within the managers folder?

Reply:
@rodrigok @graywolf336 renamed from webhook to api.